apps/settings/js/panels/date_time/panel.js

```diff
@@ -35,6 +38,12 @@ define(function(require) {
           this._elements.clockTime.textContent = DateTime.time;
         }.bind(this);
 
+        this._boundSetTimezoneInfo = function() {
+          // Only display the timezone info when auto time is enabled.
+          var info = DateTime.clockAutoEnabled ? DateTime.timezone : '';
+          this._elements.timezoneInfo.querySelector('span').textContent = info;
```

Comment:
I prefer not to do a sub-query in scripts, can we distinguish them in _elements like `this._elements.timezoneInfoContainer` and `this._elements.timezoneInfo` ?